### PR TITLE
[DPEDE-2460] Add tab content example in doc

### PR DESCRIPTION
**Pull request by emerson-pereira**
https://lumen.atlassian.net/browse/DPEDE-2460

### CODE REVIEW

File: src/documentation/pages/components-wip/tabs/examples/_horizontal-flat.vue [hunk 1]
```diff
@@ -69,33 +69,37 @@ export default class HorizontalFlat extends Vue {
...
   get codeSnippets() {
     return {
-      webcomponent: `<chi-tabs active-tab="tab-a" id="example__horizontal-flat" sliding-border></chi-tabs>
-
-${this.generateTabsContentHtml(true)}
+      webcomponent: `<chi-tabs active-tab="chi-tabs-example-tab-1" id="example__horizontal-flat" sliding-border>
+  <div slot="panels">
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-1" role="tabpanel">Tab 1 content</div>
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-2" role="tabpanel">Tab 2 content</div>
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-3" role="tabpanel">Tab 3 content</div>
```
Comment (ASanchLumen) [inline src/documentation/pages/components-wip/tabs/examples/_horizontal-flat.vue:76]:
```suggestion
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-1" role="tabpanel">Tab 1 content</div>
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-2" role="tabpanel">Tab 2 content</div>
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-3" role="tabpanel">Tab 3 content</div>
```

File: src/documentation/pages/components-wip/tabs/examples/_horizontal-solid.vue [hunk 2]
```diff
@@ -66,33 +66,37 @@ export default class HorizontalSolid extends Vue {
...
   get codeSnippets() {
     return {
-      webcomponent: `<chi-tabs active-tab="tab-a" id="example__horizontal-solid" size="lg" solid sliding-border></chi-tabs>
-
-${this.generateTabsContentHtml(true)}
+      webcomponent: `<chi-tabs active-tab="chi-tabs-example-tab-1" id="example__horizontal-solid" size="lg" solid sliding-border>
+  <div slot="panels">
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-1" role="tabpanel">Tab 1 content</div>
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-2" role="tabpanel">Tab 2 content</div>
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-3" role="tabpanel">Tab 3 content</div>
```
Comment (ASanchLumen) [inline src/documentation/pages/components-wip/tabs/examples/_horizontal-solid.vue:73]:
```suggestion
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-1" role="tabpanel">Tab 1 content</div>
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-2" role="tabpanel">Tab 2 content</div>
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-3" role="tabpanel">Tab 3 content</div>
```

File: src/documentation/pages/components-wip/tabs/examples/_vertical-flat.vue [hunk 3]
```diff
@@ -67,33 +67,37 @@ export default class VerticalFlat extends Vue {
...
   get codeSnippets() {
     return {
-      webcomponent: `<chi-tabs active-tab="tab-a" id="example__vertical-flat" vertical sliding-border></chi-tabs>
-
-${this.generateTabsContentHtml(true)}
+      webcomponent: `<chi-tabs active-tab="chi-tabs-example-tab-1" id="example__vertical-flat" vertical sliding-border>
+  <div slot="panels">
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-1" role="tabpanel">Tab 1 content</div>
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-2" role="tabpanel">Tab 2 content</div>
+    <div class="chi-tabs-panel" id="chi-tabs-example-tab-3" role="tabpanel">Tab 3 content</div>
```
Comment (ASanchLumen) [inline src/documentation/pages/components-wip/tabs/examples/_vertical-flat.vue:74]:
```suggestion
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-1" role="tabpanel">Tab 1 content</div>
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-2" role="tabpanel">Tab 2 content</div>
    <div class="chi-tabs-panel -py--3" id="chi-tabs-example-tab-3" role="tabpanel">Tab 3 content</div>
```

**Comment by ASanchLumen** [comment]
I am missing that the examples would have also been changed to use the new panels functionality. Was the scope of this task only to change the code snippets?  

I feel like having both the example and the code snippet align gives us a more stable documentation and also works for test purposes.

**Review by ASanchLumen — CHANGES_REQUESTED** [review]
Left a couple of coments. Vertical snippets do not work, we discuss in PM

**Review by josecarlosgonzalezv — APPROVED** [review]

File: src/website/views/components/tabs/_examples.pug [hunk 4]
```diff
@@ -1161,7 +1275,7 @@ h2 Examples
     | To preserve it, specify a target property on the link.
   .example.-mb--3
     .-p--3.-pb--4
-      ul#example-default-link-behavior.chi-tabs(role="tablist" aria-label="example-default-link-behavior")
+      ul#example-default-link-behavior.chi-tabs(role="tablist" aria-label="Tabs")
         li.-active
```
Comment (ASanchLumen) [inline src/website/views/components/tabs/_examples.pug:1279]: This example is missing web-component example and codesnippet

**Review by ASanchLumen — APPROVED** [review]
One example is missing, can be done in a different task

**Review by wojciechwlodarczyklumen — APPROVED** [review]

File: src/website/views/components/tabs/_examples.pug (hunk 4 above)
Comment (emerson-pereira) [reply src/website/views/components/tabs/_examples.pug]: Done

**Pull request merged**